dashboard/config/linux/bits/base.yml

```diff
@@ -171,6 +171,9 @@ config:
  # More debugging info is always good.
  - NMI_CHECK_CPU: [linux-next]
 
+ - PAGE_TABLE_CHECK: [v5.17, -s390, -arm, -timeouts_emu]
```

Comment:
I think this should go into debug.yml as it detect bugs.
base.yml mainly prepares the kernel for running in on our VMs.